compiler-rt/lib/scudo/standalone/tests/secondary_test.cpp

```diff
@@ -265,3 +265,99 @@ TEST_F(MapAllocatorWithReleaseTest, SecondaryThreadsRace) {
   Allocator->getStats(&Str);
   Str.output();
 }
+
+struct MapAllocatorCacheTest : public Test {
+  static constexpr scudo::u32 MarkerBytes = 0xDEADBEEF;
```

Comment:
Maybe a slightly better name would be UnmappedMarker. It's not really a bytes value, so it was confusing because I originally assumed it was a single byte pattern.

Reply:
Ack to this.